app/frontend/package.json

```diff
@@ -31,10 +31,11 @@
     "eject": "react-scripts eject",
     "format": "prettier --write src/",
     "lint": "eslint src/",
-    "test-ci": "CI=true react-scripts test --coverageReporters=\"text\" --coverageReporters=\"cobertura\" --coverageReporters=\"lcov\" --reporters=\"default\" --reporters=\"jest-junit\" --coverage",
+    "test-ci": "yarn tsc && CI=true react-scripts test --coverageReporters=\"text\" --coverageReporters=\"cobertura\" --coverageReporters=\"lcov\" --reporters=\"default\" --reporters=\"jest-junit\" --coverage",
     "jest": "jest --modulePaths=src",
     "storybook": "start-storybook -p 6006 -s public",
-    "build-storybook": "NODE_PATH=. build-storybook -s public"
+    "build-storybook": "NODE_PATH=. build-storybook -s public",
+    "tsc": "tsc"
```

Comment:
```suggestion
```

Reply:
I added this line so developers could do 'yarn tsc' locally using typescript version from node_modules instead of globally installed typescript version. Should I still remove it?

Reply:
I'm not sure when I'd run `tsc` separately when in VS Code tbh, so I think I'd rather not leave rarely/almost unused commands around

Reply:
I thought I would run it before committing changes, but it seems I run `yarn test-ci` instead :) I agree on not leaving rarely used commands so I shall remove it. 

Note however: running `yarn test` still won't faill if affected tests contain typescript errors. But your CI build will fail.